Sources/CNIOLinux/include/CNIOLinux.h

```diff
@@ -108,5 +108,6 @@ extern const int CNIOLinux_SO_TIMESTAMP;
 extern const int CNIOLinux_SO_RCVTIMEO;
 
 int CNIOLinux_supports_udp_segment();
+int CNIOLinux_supports_udp_gro();
```

Comment:
I only just realised these two functions have a really weird return value convention. Can we `#include <stdbool.h>` and then have these return `bool`? It'll make the rest of the code much clearer.